ui/imports/shared/popups/send/Helpers.qml

```diff
@@ -6,21 +6,28 @@ import QtQml 2.15
 import utils 1.0
 import shared.stores 1.0
 import shared.stores.send 1.0
-import AppLayouts.Wallet.stores 1.0 as WalletStores
 
 import StatusQ.Core 0.1
+import StatusQ.Core.Utils 0.1 as SQUtils
 
-import "./panels"
+import shared.popups.send.panels 1.0
 import "./controls"
 import "./views"
```

Comment:
I _think_ all of these panels/controls/views should not be needed here ideally

Reply:
You're probably right but I didn't want to spend more time on solving / improving dependencies in all the files involed! We need to do another round here, for sure.